src/System.Private.CoreLib/src/MembersMustExist.AnalyzerData

```diff
@@ -87,7 +87,6 @@ internal static extern double System.Runtime.RuntimeImports.log10(double x)
 internal static extern double System.Runtime.RuntimeImports.modf(double x, double* intptr)
 internal static extern double System.Runtime.RuntimeImports.pow(double x, double y)
 internal static extern object System.Runtime.RuntimeImports.RhBox(System.EETypePtr pEEType, void* pData)
-internal static extern void System.Runtime.RuntimeImports.RhBulkMoveWithWriteBarrier(byte* dmem, byte* smem, int size)
```

Comment:
I have checked that this is fine to do - the debugger does not actually intercepts it today.